tests/util/replica_blinker.py

```diff
@@ -0,0 +1,67 @@
+# Concord
+#
+# Copyright (c) 2019 VMware, Inc. All Rights Reserved.
+#
+# This product is licensed to you under the Apache 2.0 license (the "License").
+# You may not use this product except in compliance with the Apache 2.0 License.
+#
+# This product may include a number of subcomponents with separate copyright
+# notices and license terms. Your use of these subcomponents is subject to the
+# terms and conditions of the subcomponent's license, as noted in the LICENSE
+# file.
+
+# Add the pyclient directory to $PYTHONPATH
+
+
+import argparse
+import random
+import signal
+import subprocess
+import sys
+import time
+
+
+BLINKING_INTERVAL = (1, 10)
```

Comment:
BLINKING_INTERVAL_SECONDS?

Actually, I'd imagine the blinking to happen much more frequently, even at a sub-second rate.

In any case, we must make sure to 'blink' at least several times while processing the 300 iterations of write requests.

You could verify this by checking the logs after running the test.